install/operator/pkg/generator/assets/infrastructure/03-syndesis-server-config.yml.tmpl

```diff
@@ -64,17 +64,25 @@
         query.api.url: http://noauth-syndesis-jaeger-query:443/api
         service.name: syndesis-server
 {{- end}}
-{{- if .Productized}}
+{{- if .Syndesis.Spec.MavenRepositories}}
+      maven:
+        repositories:
+    {{- range $id, $url := .Syndesis.Spec.MavenRepositories}}
+          {{ $id }}: {{ $url }}
+    {{- end }}
+{{- else}}
+    {{- if .Productized}}
       maven:
         repositories:
           central: https://repo.maven.apache.org/maven2/
```

Comment:
Why don't we rather set a default value for `.Syndesis.Spec.MavenRepositories` so we dont have to ask every time whether is has a value or not?

Reply:
That's a better solution. Iirc default values handling is still work in progress, isn't it?

Reply:
We have this section starting here, it seems the right place to do it: https://github.com/syndesisio/syndesis/blob/9fab4e22937b38a7e18bbadbc684aa1c04488689/install/operator/pkg/syndesis/template/install.go#L146-L151